public/js/app.js

```diff
@@ -0,0 +1,34 @@
+(function() {
+
+  var app = angular.module('todoListApp', []);
+
+  app.controller('TodoList', function() {
+
+    this.items = items;
+
+    this.newItem = {};
+
+    this.addItem = function() {
+      this.items.unshift(this.newItem);
+      this.newItem = {};
+    };
+  });
+
+
+  var items = [
+
+    {content: "Welcome to TODOList!",
+     isCompleted: false
+    },
+    {content: "This item is completed!",
+     isCompleted: true
+    }
+
+
+  ];
```

Comment:
Did you intend to have these as default tasks on your todo list? In which case fine, but how would you extract this if you wanted to start with a blank todo list?